src/cljs/lambdacd/logic.cljs

```diff
@@ -10,25 +10,33 @@
 
 (defn start-ticker []
   (go-loop []
-           (re-frame/dispatch [::tick])
-           (async/<! (async/timeout poll-frequency))
-           (recur)))
+           (let [update-in-progress? @(re-frame/subscribe [::db/update-in-progress?])]
+             (when-not update-in-progress?
+               (re-frame/dispatch [::tick])
+               (async/<! (async/timeout poll-frequency))
+               (recur)))))
 
 (defn on-tick [db _]
-  (re-frame/dispatch [::update-history])
+  (when-not (:update-in-progress? db)
+    (re-frame/dispatch [::start-update-history]))
   (if (:displayed-build-number db)
```

Comment:
pipeline state still has the old behavior, right? is that intentional or just work-in-progress?